audio/rnnoise/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        xiph rnnoise 1cbdbcf1283499bbb2230a6b0f126eb9b236defd
+version             0.0.0
+revision            0
+categories          audio
+license             BSD
+maintainers         @jasonliu-- openmaintainer
```

Comment:
Customarily we request both email address and github handle to make it easier to get in touch with maintainers; see examples elsewhere in the tree.

Reply:
I was told that just putting my github handle would be fine? I don't really feel comfortable putting my personal email address in all of the portfiles.

Reply:
I'm not sure what our actual policy is on this. @ryandesign? @mojca?

Reply:
@jasonliu-- just tell them you don't care what the policy might be, you're not doing it.

Reply:
I'm not sure what the official policy is. But personally I went the same route as Jason, before becoming a member. (And so long as folks' GitHub accounts utilizes a valid e-mail address, they'll still receive notifications, etc.)

One suggestion though: surround your handle with curly braces.

Reply:
When I first started writing Portfiles, I was taught that curly braces are only needed if you want to group multiple handles for the same person, i.e.:

```
maintainers {@jverne example.org:julesverne} openmaintainer
```

results in

```
Maintainers:    GitHub: jverne, Email: julesverne@example.org
                Policy: openmaintainer
```

https://guide.macports.org/chunked/reference.html#reference.keywords.maintainers

Reply:
I'm fine with either approach.